hal/src/main/java/edu/wpi/first/hal/simulation/DriverStationDataJNI.java

```diff
@@ -77,6 +77,10 @@ public static native int registerMatchTimeCallback(
 
   public static native void setMatchTime(double matchTime);
 
+  public static native String getOpMode();
+
+  public static native void setOpMode(String opMode);
+
```

Comment:
We probably need methods to set the 2 selected values.

Reply:
Yeah, I got partway through C++, still need to do the Java/JNI layer.